installer/mac/tools/renamesha256.rb

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+# Renames `filename.tar.gz` into `filename.tar.gz-<sha256digest>`
```

Comment:
This seems like a nice, general-purpose tool. Maybe consider putting it in `$CHAIN/bin` and mark it as executable.